rkt/image_rm.go

```diff
@@ -90,6 +81,26 @@ func runRmImage(cmd *cobra.Command, args []string) (exit int) {
 		if errors > 0 {
 			stderr("rkt: %d image(s) cannot be removed", errors)
 		}
+		return fmt.Errorf("error(s) found while removing images")
```

Comment:
Since it's an error I'm returning, I leave that to the caller.

But yeah, None of the callers were adding "rkt" so I'll change that.